Fix path to headers and linker flags of docopt.cpp
author: Alex Domingo (Vrije Universiteit Brussel)
diff -Nru Applications/opensim-cmd.orig/opensim-cmd.cpp Applications/opensim-cmd/opensim-cmd.cpp
--- Applications/opensim-cmd.orig/opensim-cmd.cpp	2025-07-22 23:50:44.519787000 +0200
+++ Applications/opensim-cmd/opensim-cmd.cpp	2025-07-22 23:51:46.909712181 +0200
@@ -27,7 +27,7 @@
 #include "opensim-cmd_update-file.h"
 #include "opensim-cmd_viz.h"
 #include "parse_arguments.h"
-#include <docopt.h>
+#include <docopt/docopt.h>
 #include <iostream>
 #include <OpenSim/OpenSim.h>
 #include <OpenSim/version.h>
diff -Nru Applications/opensim-cmd.orig/opensim-cmd_info.h Applications/opensim-cmd/opensim-cmd_info.h
--- Applications/opensim-cmd.orig/opensim-cmd_info.h	2025-07-22 23:50:44.520191807 +0200
+++ Applications/opensim-cmd/opensim-cmd_info.h	2025-07-22 23:51:46.905989924 +0200
@@ -25,7 +25,7 @@
 
 #include <iostream>
 
-#include <docopt.h>
+#include <docopt/docopt.h>
 #include "parse_arguments.h"
 
 static const char HELP_INFO[] =
diff -Nru Applications/opensim-cmd.orig/opensim-cmd_print-xml.h Applications/opensim-cmd/opensim-cmd_print-xml.h
--- Applications/opensim-cmd.orig/opensim-cmd_print-xml.h	2025-07-22 23:50:44.520636000 +0200
+++ Applications/opensim-cmd/opensim-cmd_print-xml.h	2025-07-22 23:51:46.925272392 +0200
@@ -25,7 +25,7 @@
 
 #include <iostream>
 
-#include <docopt.h>
+#include <docopt/docopt.h>
 #include "parse_arguments.h"
 
 static const char HELP_PRINT_XML[] =
diff -Nru Applications/opensim-cmd.orig/opensim-cmd_run-tool.h Applications/opensim-cmd/opensim-cmd_run-tool.h
--- Applications/opensim-cmd.orig/opensim-cmd_run-tool.h	2025-07-22 23:50:44.523000000 +0200
+++ Applications/opensim-cmd/opensim-cmd_run-tool.h	2025-07-22 23:51:46.903785819 +0200
@@ -25,7 +25,7 @@
 
 #include <iostream>
 
-#include <docopt.h>
+#include <docopt/docopt.h>
 #include "parse_arguments.h"
 
 static const char HELP_RUN_TOOL[] =
diff -Nru Applications/opensim-cmd.orig/opensim-cmd_update-file.h Applications/opensim-cmd/opensim-cmd_update-file.h
--- Applications/opensim-cmd.orig/opensim-cmd_update-file.h	2025-07-22 23:50:44.523449000 +0200
+++ Applications/opensim-cmd/opensim-cmd_update-file.h	2025-07-22 23:51:46.913850637 +0200
@@ -25,7 +25,7 @@
 
 #include <iostream>
 
-#include <docopt.h>
+#include <docopt/docopt.h>
 #include "parse_arguments.h"
 
 static const char HELP_UPDATE_FILE[] =
diff -Nru Applications/opensim-cmd.orig/opensim-cmd_viz.h Applications/opensim-cmd/opensim-cmd_viz.h
--- Applications/opensim-cmd.orig/opensim-cmd_viz.h	2025-07-22 23:50:44.524594000 +0200
+++ Applications/opensim-cmd/opensim-cmd_viz.h	2025-07-22 23:51:46.918559436 +0200
@@ -24,7 +24,7 @@
  * -------------------------------------------------------------------------- */
 
 #include "parse_arguments.h"
-#include <docopt.h>
+#include <docopt/docopt.h>
 #include <iostream>
 
 #include <OpenSim/OpenSim.h>
diff -Nru Applications/opensim-cmd.orig/parse_arguments.h Applications/opensim-cmd/parse_arguments.h
--- Applications/opensim-cmd.orig/parse_arguments.h	2025-07-22 23:50:44.524969690 +0200
+++ Applications/opensim-cmd/parse_arguments.h	2025-07-22 23:51:46.921287597 +0200
@@ -23,7 +23,7 @@
  * limitations under the License.                                             *
  * -------------------------------------------------------------------------- */
 
-#include <docopt.h>
+#include <docopt/docopt.h>
 
 #include <OpenSim/OpenSim.h>
 #include <OpenSim/Moco/osimMoco.h>
diff -Nru Applications/opensim-cmd.orig/CMakeLists.txt Applications/opensim-cmd/CMakeLists.txt
--- Applications/opensim-cmd.orig/CMakeLists.txt	2025-07-23 00:09:32.223683968 +0200
+++ Applications/opensim-cmd/CMakeLists.txt	2025-07-23 00:09:43.886753358 +0200
@@ -20,7 +20,7 @@
             parse_arguments.h
     )
 
-target_link_libraries(opensim-cmd docopt_s osimMoco)
+target_link_libraries(opensim-cmd docopt osimMoco)
 
 if(BUILD_TESTING)
     subdirs(test)
